bears/scss/SCSSLintBear.py

```diff
@@ -268,9 +268,9 @@ def generate_config(filename, file,
                                    'width': indent_size},
                    'LeadingZero': {
                                    'enabled': True,
-                                   'style': (
-                                       'exclude_zero' if exclude_leading_zero
-                                        else 'include_zero')
```

Comment:
this should be fixed by removing one space from before the `else`.  No need to modify the line above it.